sklearn/utils/_encode.py

```diff
@@ -40,13 +47,23 @@ def _unique_python(values, *, return_inverse):
                        for t in set(type(v) for v in values))
         raise TypeError("Encoders require their input to be uniformly "
                         f"strings or numbers. Got {types}")
+    ret = (uniques, )
 
     if return_inverse:
         table = {val: i for i, val in enumerate(uniques)}
         inverse = np.array([table[v] for v in values])
-        return uniques, inverse
+        ret += (inverse, )
+
+    if return_counts:
+        uniques_dict = Counter(values)
```

Comment:
this seems duplicated with `_get_counts`?

Reply:
I could move `_get_counts` here.

At this point, we know the dtype is object. And it is only two lines of code?